packages/frontend/editor-ui/src/stores/folders.store.ts

```diff
@@ -292,13 +292,13 @@ export const useFoldersStore = defineStore(STORES.FOLDERS, () => {
 
 					// Add all descendants of this child
 					if (child.children?.length) {
-						childResult.push(...processFolderWithChildren(child).slice(1));
+						childResult.push.apply(childResult, processFolderWithChildren(child).slice(1));
```

Comment:
This and the next one were automatically changed based on this eslint rule that we've recently added `n8n-local-rules/no-argument-spread`

Reply:
odd rule to have 🤔